File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        ans=0 
        for j in range(len(strs[0])):
            for i in range(1,len(strs)): 
                if strs[i][j]<strs[i-1][j]: 
                    ans+=1 
                    break 
        return ans                 
    

File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.cpp
class Solution {
public:
    int minDeletionSize(vector<string>& strs) {
        int n=strs.size(); 
        int m=strs[0].size(); 
        int cnt=0; 
        for(int j=0;j<m;j++){
            for(int i=0;i<n-1;i++){    //cols
                if(strs[i][j]>strs[i+1][j]){    //rows 
                    cnt++; 
                    break; 
                }
            }
        }
        return cnt; 
    }
};